testdata/baselines/reference/submodule/conformance/callbackTagVariadicType.errors.txt

```diff
@@ -1,16 +1,20 @@
-callbackTagVariadicType.js(9,18): error TS2554: Expected 1 arguments, but got 2.
+callbackTagVariadicType.js(3,4): error TS2370: A rest parameter must be of an array type.
+callbackTagVariadicType.js(9,13): error TS2345: Argument of type '["a", "b"]' is not assignable to parameter of type 'string'.
 
 
-==== callbackTagVariadicType.js (1 errors) ====
+==== callbackTagVariadicType.js (2 errors) ====
     /**
      * @callback Foo
      * @param {...string} args
+       ~~~~~~~~~~~~~~~~~~~~~~~

```

Comment:
this shows the old, still unsupported semantics where the parameter `args` becomes a rest parameter *and* the type `string` becomes `string[]`. Only the first happens now, based on seeing how a real person (Titus Woormer) wrote overloads.